test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/MiddlewareAnalysisSampleTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
It looks like most of what this sample is testing is logging stuff, which is already covered in the other tests, do we still need this one?